packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -68,9 +67,9 @@ class TransactionService {
   }
 
   createTransaction(name, type, options) {
-    var tr = new Transaction(name, type, options)
+    const tr = new Transaction(name, type, options)
     this.setCurrentTransaction(tr)
-    if (options.checkBrowserResponsiveness) {
+    if (type !== PAGE_LOAD && options.checkBrowserResponsiveness) {
```

Comment:
I don't think we should add a check for page load here. we should make sure that we set `options. checkBrowserResponsiveness` on page loads transactions.

Reply:
Not really, we should not start the counter from the first. right now we do start the timer which starts a timer for no reason and then disable it on post-processing. We should handle it here and do not even care on the end.